Doc/glossary.rst

```diff
@@ -134,6 +134,16 @@ Glossary
       iterator's :meth:`~object.__anext__` method until it raises a
       :exc:`StopAsyncIteration` exception.  Introduced by :pep:`492`.
 
+   atomic operation
+      An operation that completes as a single indivisible unit without
+      interruption from other threads.  Atomic operations are critical for
+      :term:`thread-safe` programming because they cannot be observed in a
+      partially completed state by other threads.  In the
+      :term:`free-threaded <free threading>` build, elementary operations
```

Comment:
Elementary operations is vague term and not properly defined. For example `list.append` and `set.add` both may be considered elementary however only the former is atomic in all cases.